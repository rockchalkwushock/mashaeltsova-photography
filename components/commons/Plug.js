import PropTypes from 'prop-types'

import P from './P'
import { styles } from '../../lib'

const Plug = ({ messages }) => (
  <div>
    <P className="plug">{messages.sectionPlug}</P>
    <style jsx>{`
      div {
        background-color: ${styles.colors.peach};
        font-family: ${styles.fonts.courgette};
        padding: 1em;
      }
    `}</style>
  </div>
)

Plug.propTypes = {
  messages: PropTypes.object.isRequired
}

export default Plug
