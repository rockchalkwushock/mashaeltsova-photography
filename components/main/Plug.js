import { styles } from '../../lib'

const Plug = () => (
  <div>
    <p>
      Contact me if you want to relive those precious moments for years to come!
    </p>
    <style jsx>{`
      div {
        background-color: ${styles.colors.peach};
        font-family: ${styles.fonts.courgette};
        padding: 1em;
      }
    `}</style>
  </div>
)

export default Plug
