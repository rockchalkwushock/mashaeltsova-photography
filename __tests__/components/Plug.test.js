import React from 'react'
import { render } from 'enzyme'

import Plug from '../../components/main/Plug'

describe('Component: Plug', () => {
  test('Renders without exploding', () => {
    const tree = render(<Plug />)
    expect(tree).toMatchSnapshot()
  })
})
