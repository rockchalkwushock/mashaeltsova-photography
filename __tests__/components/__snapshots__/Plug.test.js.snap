// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Plug Renders without exploding 1`] = `
<div
  class="jsx-3428991023"
>
  <p
    class="jsx-3428991023"
  >
    Contact me if you want to relive those precious moments for years to come!
  </p>
</div>
`;
